projects/grpc-httpjson-transcoding/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -eu
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# This project uses bazel rules_fuzzing. Script copied from OSS fuzz's bazel-rules-fuzzing-test.
+# https://github.com/google/oss-fuzz/blob/62fce2a587b5d0eff941c392b343edfc68eb4069/projects/bazel-rules-fuzzing-test/build.sh
+
+declare -r QUERY='
+    let all_fuzz_tests = attr(tags, "fuzz-test", "//...") in
+    $all_fuzz_tests - attr(tags, "no-oss-fuzz", $all_fuzz_tests)
+'
+
+declare -r PACKAGE_SUFFIX="_oss_fuzz"
+declare -r OSS_FUZZ_TESTS="$(bazel query "${QUERY}" | sed "s/$/${PACKAGE_SUFFIX}/")"
+
+bazel build -c opt --config=oss-fuzz --linkopt=-lc++ \
```

Comment:
i might be missing this but is `--config=oss-fuzz` meaningful?

(Envoy's are defined here https://github.com/envoyproxy/envoy/blob/ca1455ccf81ca4cd742903361d7c2a94f37f5be0/.bazelrc#L301-L314)

Reply:
AH! just saw the PR description. Resolved :)

Reply:
Yup! We are following the docs linked here: https://github.com/bazelbuild/rules_fuzzing#oss-fuzz-integration